src/test/java/uk/gov/companieshouse/api/accounts/transformer/DirectorsApprovalTransformerTest.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.companieshouse.api.accounts.transformer;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.api.accounts.model.entity.directorsreport.DirectorsApprovalDataEntity;
+import uk.gov.companieshouse.api.accounts.model.entity.directorsreport.DirectorsApprovalEntity;
+import uk.gov.companieshouse.api.accounts.model.rest.directorsreport.DirectorsApproval;
+
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class DirectorsApprovalTransformerTest {
```

Comment:
Should we assert the `date` field is mapped in this test class too?

Reply:
Added the code related to date assertion in directors approval transformer class